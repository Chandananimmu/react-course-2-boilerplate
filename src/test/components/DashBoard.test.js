import React from "react";
import { shallow } from "enzyme";
import DashBoardPage from "../../components/DashBoardPage";
test("shud render DashBoard correctly", () => {
  const wrapper = shallow(<DashBoardPage />);
  expect(wrapper).toMatchSnapshot();
});
