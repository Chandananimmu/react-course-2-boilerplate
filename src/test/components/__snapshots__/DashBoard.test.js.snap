// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shud render DashBoard correctly 1`] = `
<div>
  Dashboard Page content
</div>
`;
